sources/procfs.go

```diff
@@ -455,7 +409,14 @@ func parseReadWriteBytes(statsFile string, helpText string, promName string) (me
 	if err != nil {
 		return nil, err
 	}
-	metricList = append(metricList, lustreStatsMetric{title: promName, help: helpText, value: result})
+	l := lustreStatsMetric{
+		title:           promName,
+		help:            helpText,
+		value:           result,
+		extraLabel:      "operation",
```

Comment:
extraLabel is what we use to figure out what the label should be called. I could hard code it I suppose, but I try to avoid that.